civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -3,9 +3,76 @@
 
 /* Welcome page */
 .page-welcome-page {
-  .pane-user-login {
-    max-width: 438px;
-    margin: 0 auto;
-    width: 100%;
-  }
-}
\ No newline at end of file
+    .pane-user-login {
+        max-width: 438px;
+        margin: 0 auto;
+        width: 100%;
+    }
+}
+
+.page-reports {
+    .content {
+        padding: 0;
+        background-color: transparent;
+    }
+
+    .chr_panel__footer {
```

Comment:
I'm not sure you still need to customize this element. Have you taken a look at the "My task" panel on the dashboard?

<img width="800" alt="chr_panel--no-padding" src="https://cloud.githubusercontent.com/assets/6400898/15066785/2e85cb14-136c-11e6-8754-d451769b5a59.png">

That seems pretty similar to the [design](https://projects.invisionapp.com/share/EK3ZEW4ZJ#/screens/148828068) you need to implement (one header, one content without padding, a button section on the bottom)

If you inspect that panel you will see that it implements a variation of the `.chr_panel` class, called `.chr_panel--no-padding` which allows the content to go full width, while still keeping the padding for the header and footer

Please give it a look and see if you can reuse that without customizing anything else